### PR TITLE
add push-image-frontend and push-image-backend

### PR DESCRIPTION
**Pull request by gregoriusjimmy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

docker image build still manual

## What is the new behavior?

automate docker image build with github workflow
## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -27,3 +34,63 @@ jobs:
...
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: ./deployment/frontend.yaml
```
Comment (williammfu) [inline .github/workflows/CI.yml:93]: Kayanya `context` disini buat path frontend/backend nya deh misal `./frontend`, CMIIW ya

Comment (gregoriusjimmy) [reply .github/workflows/CI.yml:93]: iya tapi itu kok gakedetect ya, apa harus ../../deployment/frontend.yaml ?

Comment (gregoriusjimmy) [reply .github/workflows/CI.yml:93]: oh bukan buat yaml filenya yak?

Comment (williammfu) [reply .github/workflows/CI.yml:93]: keanya Dockerfile jim

Comment (gregoriusjimmy) [reply .github/workflows/CI.yml:93]: oh iya yaml kan buat kubectl :v

**Review by lazuardyk — APPROVED** [review]

**Review by rizkybaihaqy — APPROVED** [review]

**Pull request merged**